distribution/docs/src/main/jdocs/content/_plugins/pre-resource-plugin-intro-contents.adoc

```diff
@@ -0,0 +1,16 @@
+:type: pluginIntro
+:status: published
+:title: Pre-Resource Plugins
+:link: _pre_resource_plugins
+:summary: Perform any changes to a resource associated with a metacard prior to download.
+:plugintypes: preresource
+:order: 09
+
+Pre-Resource plugins are invoked before a request to retrieve a resource is sent to a Source. 
+This is an opportunity to take any action on the request, including but not limited to:
```

Comment:
❓ Is the action taken on the request or the resource? This section seems to use them interchangeably.